packages/@o3r/rules-engine/src/engine/operator/operators/date-based.operators.ts

```diff
@@ -16,6 +16,40 @@ export const inRangeDate: Operator<Date, [DateInput, DateInput], DateInput> = {
   validateRhs: isValidDateRange
 };
 
+/**
+ * Check if the value of the variable is in the next x minutes
+ *
+ * @title is in next minutes
+ */
+export const dateInNextMinutes: Operator<Date, [DateInput, number], DateInput> = {
+  name: 'dateInNextMinutes',
+  evaluator: (leftDateInput, [rightDate, minutes]) => {
+    const targetDate = new Date(rightDate);
+    const leftDate = new Date(leftDateInput);
+    targetDate.setMinutes(targetDate.getMinutes() + +minutes);
+    return leftDate <= targetDate;
```

Comment:
maybe it would be better to reuse `inRangeDate`
```typescript
evaluator: (leftDateInput, [rightDate, minutes]) => {
  return inRangeDate.evaluator(leftDateInput, [rightDate, new Date(rightDate).getTime() + minutes * 60000)]);
}
```